### ISSUE TITLE
Deprecated alias in use HomeAssistantType

### ISSUE THREAD

**Issue by lcollins** [albertogeniola/meross-homeassistant#499]
**Describe the bug**
A deprecated alias is currently in use (HomeAssistantType)

**Your environment**
HomeAssistant version: 2024.6.0.dev202405030217
Hassio Version (if applicable): Home Assistant OS 12.3.rc2
Hardware environment: RaspberryPI4 (32bit)

**Logs taken when the issue happened**
```
2024-05-03 12:03:50.584 WARNING (ImportExecutor_0) [homeassistant.helpers.typing] HomeAssistantType was used from meross_cloud, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'meross_cloud' custom integration
2024-05-03 12:03:50.598 WARNING (ImportExecutor_0) [homeassistant.helpers.typing] HomeAssistantType was used from meross_cloud, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'meross_cloud' custom integration
2024-05-03 12:03:50.611 WARNING (ImportExecutor_0) [homeassistant.helpers.typing] HomeAssistantType was used from meross_cloud, this is a deprecated alias which will be removed in HA Core 2025.5. Use homeassistant.core.HomeAssistant instead, please report it to the author of the 'meross_cloud' custom integration
```

**Comment by albertogeniola** [comment]
Addressed into new beta version. Will be merged into v1.3.2 as soon as the beta is over. 

Thank you for reporting!

**Comment by lcollins** [comment]
Many thanks @albertogeniola, I can confirm the issue is resolved with 1.3.2
